close_all_positions.py

```
@@ -0,0 +1,105 @@
+    headers = {"X-MBX-APIKEY": binance['api_key']}
+
+    timestamp = int(time.time() * 1000)
+    query_string = f"timestamp={timestamp}"
+    signature = hmac.new(
+        binance['api_secret'].encode('utf-8'),
+        query_string.encode('utf-8'),
+        hashlib.sha256
+    ).hexdigest()
+
+    full_url = f"{url}?{query_string}&signature={signature}"
+    response = requests.get(full_url, headers=headers)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)